docs/manuals/auth/authentication.md

```diff
@@ -43,90 +43,75 @@ Taipy Enterprise edition supports three authentication protocols:
       default authenticator.
     - When `login()^` is called:
 
-        - A default authenticator is created if there is none, based on files
-          that may be sitting in the current directory or the application's
-          [Global Configuration](../core/config/global-config.md).
+        - A default authenticator is created if there is none, based on the authentication configuration.
         - If Taipy could not find the configuration allowing to create such a default
-          authenticator, a *None* authenticator is created and set as the default 
+          authenticator, a *None* authenticator is created and set as the default
           authenticator.
         - The default authenticator's `(Authenticator.)login()^` function is invoked
           with the arguments that were provided to `login()^`.
-    
+
 
 Beside their specific parameters, all authenticators have two parameters that you
 can provide in the [`Authenticator` constructor](Authenticator.__init__()^):
 
 - *secret_key*: a string that is used to encrypt the user credentials. Because
-  credentials are transmitted back and fourth when running Taipy on a server
+  credentials are transmitted back and forth when running Taipy on a server
   (when a REST or a GUI application runs), this information is encrypted. You
   can provide an encryption key, or let Taipy create one for you.
 - *auth_session_duration*: how long in seconds should the credentials created
-  by this authenticator be considered valid. 
+  by this authenticator be considered valid.
 
-!!! Note "Global configuration"
-    If you are using the Taipy Configuration to configure the Core features (Scenarios, Task
-    scheduling, Datanodes, ...), these parameters can also
-    be set in the [Taipy Global Configuration](../core/config/global-config.md).
+## 'None' Authenticator
 
-    The global configuration properties related to the default authentication
-    are used if the parameters are not loaded from a
-    *taipy&#x5F;auth&#x5F;&lt;protocol&gt;.json* file.
+The *None* authenticator does not check for user declaration or password match.
+It is designed so that developers can start building secure applications before
+the actual authentication system and processes are defined or deployed.
 
-    Here is how you can use the _**Config.global_config.secret_key**_ and
-    _**Config.global_config.auth_session_duration**_ properties of the
-    global configuration:
+When the *None* authenticator `(Authenticator.)login()^` method is called, it always
+returns a valid `Credentials^` object, no matter what username and password
+are provided.
+
+To create a *None* authenticator, you can instantiate an `Authenticator^` object
+setting the *protocol* argument of the constructor to "none".
+
+!!! Note "Using Taipy configuration to set default authenticator"
+    To set the *default authenticator* to `none` you can use the authentication configuration either in Python or Toml:
```

Comment:
Make sure you use TOML, or toml, or Toml appropriately, and everywhere.
To me, and because it's an acronym, it should *always* be capitalized: TOML.

Reply:
I agree.